### PR TITLE
Added baserow.io in DBaaS

### PR DESCRIPTION
**Pull request by dilip-borad**
Thanks in advance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1031,6 +1031,7 @@ Table of Contents
    * [Macrometa](https://www.macrometa.com/) - a noSQL database, Pub/Sub, event processing, and serverless edge computing platform for building geo-distributed and real-time applications. Free dev account gives access to 10,000 Operations/Day & 200MB Storage.
    * [Planetscale](https://planetscale.com/) - PlanetScale is a MySQL compatible, serverless database platform powered by Vitess, 3 databases for free with 10GB storage, 100 Million rows read/mo per database, and 10 Million rows written/mo per database.
    * [YugabyteDB](http://cloud.yugabyte.com) - YugabyteDB is a distributed SQL database compatible with PostgresSQL. The cloud free tier is a 2 vCPU, 4GB RAM, 10GB Disk.
+   * [Baserow](https://baserow.io/) — Baserow is an open source no-code database and Airtable alternative. Create your own database without technical experience. Our user friendly no-code tool gives you the powers of a developer without leaving your browser.
```
Comment (ripienaar) [inline README.md:1034]: Thanks, what does "Free for now" mean on the pricing page? That's not enough for me to go on to determine if it should be added here.  Based on the blow (found top of the list), I need to know a bit more.

```
NOTE: This list is only for as-a-Service offerings, not for self-hosted software. For a service to be eligible it has to offer a free tier and not just a free trial. If the free tier is time-bucketed it has to be for at least a year. 
```

Comment (dilip-borad) [reply README.md:1034]: No problem at all.
Thanks for informing me of this valuable information.

**Comment by AriESQ** [comment]
Can you insert this alphabetically in the list please.

**Comment by winzsys** [comment]
MERGING TO [winazys]

**Pull request closed without merging**